data/device-tests/devices/dell-wd19tb-ec.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "Dell WD19TB Dock (EC)",
+  "guids": [
+    "cd357cf1-40b2-5d87-b8df-bb2dd82774aa"
+  ],
+  "releases": [
+    {
+      "version": "01.00.00.04",
+      "file": "0f7990c89255596d6258c9715b59a40878d37c0ac3ef368aa01c99bed0c59a33-WD19FirmwareUpdateLinux_01.00.15.cab"
```

Comment:
Hmm this is actually leaking the full filename; IE someone could easily just prepend https://fwupd.org/downloads/.  Could you just make a local symlink for now to obfuscate it?

Reply:
Fixed. If you run that test you'll have to download it manually and rename it too.

Reply:
OK thanks.  When it goes stable we can revert it back to this too.